docs/releases/2.0.txt

```diff
@@ -177,6 +177,9 @@ Management Commands
 * :djadmin:`inspectdb` now translates MySQL's unsigned integer columns to
   ``PositiveIntegerField`` or ``PositiveSmallIntegerField``.
 
+* :djadmin:`makemessages` now accepts the optional argument ``--add-location``
+  for ``gettext`` 0.19 or newer.
```

Comment:
@francoisfreitag  Updated. Thanks for the suggestion, and also for the usage of ` and ``!

https://github.com/django/django/pull/8333/commits/0ba6e2e76ac3bc26f23d8baf9f4270d248ff3c69#diff-af003fcfed7cfbdeb396f8647ed0f92fR180